integrations/rust/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "scalar_api_reference",
+  "name": "@scalar/rust-api-reference",
```

Comment:
ah, my convention is to use the name we publish the package, in that case the crate is called scalar_api_reference. that way, we avoid to make up another name (@scalar/rust-api-reference)